### DISCUSSION TITLE
Line chart with dots

### DISCUSSION THREAD

**Discussion by joseplca** [patrykandpatrick/vico#522] [Questions] [answered]
Is there any chance to draw a line with a dot pattern?

Something like this:

![image](https://github.com/patrykandpatrick/vico/assets/118980847/5a707823-4801-4850-acba-afb3b6d800fd)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! There’s no such feature yet, but it’s a planned addition. Please see #469.
